src/analytics/internal/analytics.js

```diff
@@ -1,14 +1,11 @@
 import { EVENT_TYPES } from './constants'
+import sendToServer from './send-to-server'
```

Comment:
This might be better passed into the constructor as well, so we can easily mock it out.